src/cheri/csv/CHERI_CSR.csv

```diff
@@ -1,48 +1,48 @@
 Always update the CSR with <<SCADDR>> even if the address didn't change, including the MODE field in the address for simplicity.
 Vector range check ^*^ if vectored mode is programmed.","✔","","","S-mode","Supervisor Trap-Vector Base-Address Capability","","","","","","","","","","","","","","","","","","","","",""
-"sscratch_y","0x140","sscratch","S","SRW, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","S-mode","Supervisor Scratch Capability","","","","","","","","","","","","","","","","","","","","",""
-"sepc_y","0x141","sepc","S","SRW, <<asr_perm>>","<<root-rx-cap>>","Apply <<section_invalid_addr_conv>>.
+"sscratch_y","0x140","sscratch","S","SRW, <<asr_perm>>","tag=0, otherwise specified by the platform","Update the CSR using <<SCADDR>>.","direct write","","","","S-mode","Supervisor Scratch Capability","","","","","","","","","","","","","","","","","","","","",""
+"sepc_y","0x141","sepc","S","SRW, <<asr_perm>>","Nominally <<root-rx-cap>>","Apply <<section_invalid_addr_conv>>.
 Always update the CSR with <<SCADDR>> even if the address didn't change.","Apply <<section_invalid_addr_conv>> and update the CSR with the result if the address changed,
 direct write if address didn't change","✔","","✔","S-mode","Supervisor Exception Program Counter Capability","","","","","","","","","","","","","","","","","","","","",""
-"vstvec_y","0x205","vstvec","VS","HRW, <<asr_perm>>","<<root-rx-cap>>","Apply <<section_invalid_addr_conv>>.
+"vstvec_y","0x205","vstvec","VS","HRW, <<asr_perm>>","specified by the platform","Apply <<section_invalid_addr_conv>>.
 Always update the CSR with <<SCADDR>> even if the address didn't change, including the MODE field in the address for simplicity.
 Vector range check ^*^ if vectored mode is programmed.","Apply <<section_invalid_addr_conv>>.
 Always update the CSR with <<SCADDR>> even if the address didn't change, including the MODE field in the address for simplicity.
 Vector range check ^*^ if vectored mode is programmed.","✔","","","H","Virtual Supervisor Trap-Vector Base-Address Capability","","","","","","","","","","","","","","","","","","","","",""
-"vsscratch_y","0x240","vsscratch","VS","HRW, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","H","Virtual Supervisor Scratch Capability","","","","","","","","","","","","","","","","","","","","",""
-"vsepc_y","0x241","vsepc","VS","HRW, <<asr_perm>>","<<root-rx-cap>>","Apply <<section_invalid_addr_conv>>.
+"vsscratch_y","0x240","vsscratch","VS","HRW, <<asr_perm>>","tag=0, otherwise specified by the platform","Update the CSR using <<SCADDR>>.","direct write","","","","H","Virtual Supervisor Scratch Capability","","","","","","","","","","","","","","","","","","","","",""
+"vsepc_y","0x241","vsepc","VS","HRW, <<asr_perm>>","specified by the platform","Apply <<section_invalid_addr_conv>>.
```

Comment:
```suggestion
"vsepc_y","0x241","vsepc","VS","HRW, <<asr_perm>>","tag=0, otherwise specified by the platform","Apply <<section_invalid_addr_conv>>.
```